homework/homework20/test_library.py

```diff
@@ -0,0 +1,177 @@
+"""This program covers the "Library" program with tests"""
+
+import unittest
+import datetime
+from loguru import logger
+from homework.homework11.library import Books, User
+
+# Configure logger
+current_time = datetime.datetime.now().strftime('%Y-%m-%d_%H-%M-%S')
+logger.add(f'logs/{current_time}_test_library.log',
+           format="{time:YYYY-MM-DD HH:mm:ss} | {level} | "
+                  "{file}:{line} | {message}",
+           level='INFO')
```

Comment:
You can use 1 logger across a single application, e.g.: https://stackoverflow.com/questions/75489656/how-to-use-a-single-python-logging-object-throughout-the-python-application